crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/controller/ContributionControllerTest.java

```diff
@@ -14,38 +14,45 @@
 import uk.gov.justice.laa.crime.contribution.data.builder.TestModelDataBuilder;
 import uk.gov.justice.laa.crime.contribution.dto.CalculateContributionDTO;
 import uk.gov.justice.laa.crime.contribution.exeption.ValidationException;
+import uk.gov.justice.laa.crime.contribution.model.ApiContributionTransferRequest;
+import uk.gov.justice.laa.crime.contribution.model.ApiMaatCalculateContributionRequest;
+import uk.gov.justice.laa.crime.contribution.model.ApiMaatCalculateContributionResponse;
 import uk.gov.justice.laa.crime.contribution.model.common.ApiContributionSummary;
-import uk.gov.justice.laa.crime.contribution.model.maat_api.ApiMaatCalculateContributionRequest;
-import uk.gov.justice.laa.crime.contribution.model.maat_api.ApiMaatCalculateContributionResponse;
+import uk.gov.justice.laa.crime.contribution.service.ContributionService;
 import uk.gov.justice.laa.crime.contribution.service.MaatCalculateContributionService;
 import uk.gov.justice.laa.crime.contribution.validation.CalculateContributionValidator;
 
 import java.util.List;
 import java.util.Optional;
 
 import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static uk.gov.justice.laa.crime.contribution.util.RequestBuilderUtils.buildRequestGivenContent;
 
 @DirtiesContext
 @AutoConfigureMockMvc(addFilters = false)
-@WebMvcTest(CrownCourtContributionController.class)
-class CrownCourtContributionControllerTest {
+@WebMvcTest(ContributionController.class)
+class ContributionControllerTest {
 
-    private static final String ENDPOINT_URL = "/api/internal/v1/contribution/calculate-contribution";
+    private static final String BASE_URL = "/api/internal/v1/contribution";
+    private static final String ENDPOINT_URL = BASE_URL + "/calculate-contribution";
+    private static final String GET_CONTRIBUTION_SUMMARIES_ENDPOINT_URL =
+            BASE_URL + "/" + TestModelDataBuilder.REP_ID + "/summaries";
```

Comment:
Trivial - if the BASE_URL is defined as "/api/internal/v1/contribution/" we can avoid one concatenation.
BASE_URL + TestModelDataBuilder. REP_ID